### PR TITLE
[MNT] Deprecate non-federated query tool config, bump Docker Compose required ver.

### LINKED ISSUE #204: Make it clearer that only node-API should be listed in `local_nodes.json`
### Description

@barbarastrasser mentioned that from the docs it was not clear enough that `local_nodes.json` should only contain the addresses of node-APIs, and not any federation-APIs.

If you get this wrong, bad things will happen: https://github.com/neurobagel/federation-api/issues/89

### Relevant documentation pages

_No response_

### Additional details

_No response_

### LINKED ISSUE #213: Remove mention of non-federated query tool installation
Need to update:
- [x] getting started page
- [x] deployment profiles
- [x] API usage page

### LINKED ISSUE #215: Required version of Docker Compose is too old for secrets
### Description

We currently specify a minimum Docker Compose version of 2.0.0, but this version actually (apparently) doesn't support Docker secrets set by environment variables.

## Steps to reproduce
If you try to spin up a local node using Docker Compose following our docs with a version around 2.0.0 (e.g., 2.2.3), you will get an error on launch:
```bash
secrets.db_admin_password Additional property environment is not allowed
```

### Steps to fix
Based on https://docs.docker.com/compose/release-notes/, it looks like support for environment secrets, including during the build step, was introduced in v2.7.0. We need to update the minimum Docker Compose version to match.

### Relevant documentation pages

http://127.0.0.1:8000/getting_started/#docker-and-docker-compose

### Additional details

_No response_

### PR DESCRIPTION
**Pull request by alyssadai**
- Closes #213 
- Closes #215 
- Closes #204 

Changes proposed in this pull request:

- Query tool is now specified to point only to an f-API instance
- Minor formatting edits for readability
- Minimum Docker Compose version updated to one that supports secrets via environment vars

## Checklist
_Please leave checkboxes empty for PR reviewers_

- [x] PR has an interpretable title with a prefix (`[ENH]`, `[FIX]`, `[REF]`, `[TST]`, `[CI]`, `[MNT]`, `[INF]`) _see our [Contributing Guidelines](https://neurobagel.org/contributing/CONTRIBUTING/#pull-request-guidelines) for more info)_
- [x] PR links to GitHub issue with mention `Closes #XXXX`
- [x] Checks pass

### CODE REVIEW

File: docs/config.md [hunk 1]
```diff
@@ -90,7 +89,7 @@ At minimum, we recommend reviewing and changing the values of the following vari
 > `NB_GRAPH_ADMIN_PASSWORD`  
 > `NB_GRAPH_USERNAME`  
 > `NB_GRAPH_PASSWORD`  
-> `NB_GRAPH_DB` 
+> `NB_GRAPH_DB`
```
Comment (surchs) [inline docs/config.md:92]:
```suggestion
> `NB_GRAPH_DB`
```
accidental space added?

**Review by surchs — APPROVED** [review]
one comment  -> :cook:

**Comment by alyssadai** [comment]
Hey @surchs, just to wrap up a small stale issue into this PR, I added a commit https://github.com/neurobagel/documentation/pull/217/commits/c92a4ab3819fdd00825c63c84cce06d694253ad2 to address Barbara's request for more clarity concerning `local_nb_nodes.json` https://github.com/neurobagel/documentation/issues/204.

Let me know if you think the addition is clear enough that `local_nb_nodes.json` should only include n-APIs.

**Comment by surchs** [comment]
> Hey @surchs, just to wrap up a small stale issue into this PR, I added a commit [c92a4ab](https://github.com/neurobagel/documentation/commit/c92a4ab3819fdd00825c63c84cce06d694253ad2) to address Barbara's request for more clarity concerning `local_nb_nodes.json` #204 (since I'm editing the config page anyways).
> 
> Let me know if you think the addition is clear enough that `local_nb_nodes.json` should only include n-APIs.

Yep, that is a good way to address it I think. Thanks for that!

**Pull request merged**